packages/design-system/src/components/Spinner/index.ts

```diff
@@ -1 +1 @@
-export { default as Spinner } from './Spinner';
+export { default as Spinner, SpinnerProps } from './Spinner';
```

Comment:
We weren't exporting `SpinnerProps` for some reason and I added it here so I could use it in the 🚾